tools/tcprtt.py

```diff
@@ -186,14 +212,22 @@
 print("Tracing TCP RTT... Hit Ctrl-C to end.")
 
 def print_section(addr):
-    if args.byladdr:
-        return inet_ntop(AF_INET, struct.pack("I", addr)).encode()
-    elif args.byraddr:
-        return inet_ntop(AF_INET, struct.pack("I", addr)).encode()
+    addrstr = "*******"
+    if (addr):
+        addrstr = inet_ntop(AF_INET, struct.pack("I", addr)).encode()
+
+    avglat = ""
+    if args.avg:
+        lats = b.get_table("latency")
+        lat = lats[ctypes.c_ulong(addr)]
+        avglat = " [AVG %d]" % (lat.latency / lat.count)
```

Comment:
I would like average latency to be printed as a footer after histogram, similar to biolatency.py.

Reply:
It 's a little different from biolatency. biolatency show a summary info for all the disks as a footer. (maybe also need a per-disk info)
I'd like tcprtt to show average RTT of each histogram(--laddr/--raddr), but section_print_fn could print additional info in header only. And for each of host-to-host(--laddr/--raddr) histogram, showing extension behind address (Ex, Remote Addres = 10.194.87.90 [AVG 2188]) seems clear.

Reply:
Okay. I am fine with the info just besides the address.